src/AppInstallerCommonCore/Manifest/ManifestYamlPopulator.cpp

```diff
@@ -249,6 +263,23 @@ namespace AppInstaller::Manifest
                     std::move(v1InstallerFields.begin(), v1InstallerFields.end(), std::inserter(result, result.end()));
                 }
             }
+
+            if (manifestVersion >= ManifestVer{ s_ManifestVersionV1_1 })
+            {
+                std::vector<FieldProcessInfo> fields_v1_1 =

```

Comment:
I don't see anywhere that we are limiting what the server will accept.  Letting manifests into winget-pkgs when the feature behind them has not yet been implemented will likely lead to failures.  Either now because someone relies on it for a package and it doesn't actually do anything or in the future when we start using it and find that we wanted validation to prevent some case that is now already present.

It feels like we should have:
1. Add new version to client to allow for local testing; full validation fails if any 1.1 fields are present even if manifest is 1.1
2. As we add functionality and validation, we start allowing fields that are ready

Reply:
Evaluated 1 by 1 offline and we're ok with them for now